put-iam-role.yaml

```diff
@@ -0,0 +1,42 @@
+---
+platform: linux
+
+image_resource:
+  type: docker-image
+  source:
+    repository: governmentpaas/awscli
+
+inputs:
+  - name: common-tasks
+  - name: source
+
+outputs:
+- name: output
+
+params:
+  AWS_ACCESS_KEY_ID: ((role-putter-access-key-id))
+  AWS_SECRET_ACCESS_KEY: ((role-putter-secret-access-key))
+  AWS_NODES_ROLE_ARN: ((node-arn))
+  APP_NAME: ((app-name))
+
+run:
+  path: sh
+  args:
+  - -ecx
+  - |
+    sed 's@${AWS_NODES_ROLE_ARN}@'$AWS_NODES_ROLE_ARN'@g' ./common-tasks/templates/trust-policy.json.tmpl > ./trust-policy.json
```

Comment:
I think `sed` in this case is reasonably simple and not worth the effort to retrieve/use something else like `envsubst` if not already available in the Docker image.